internal/grpcapi/order_service.go

```diff
@@ -58,7 +57,7 @@ func (s OrderService) InsertOrder(ctx context.Context, order *pb.Order) (*pb.Ord
 
 		return nil, status.Error(codes.Internal, "init tx failed") //nolint:wrapcheck // should be wrapped as is
 	}
-	err = db.PrepareTransaction(ctx, transaction, txID)
+	err = db.PrepareTransaction(ctx, transaction, txID.String())
 	if err != nil {
 		logger.Error("prepare tx failed ", err)
```

Comment:
logrus has a special method to log errors. logger.WithError(err) - this is a correct way for the structured logging.

Reply:
done